tests/integration/knowledge_bases/test_kb_sql.py

```diff
@@ -400,55 +393,3 @@ def test_metadata(self):
         assert 'noise' in ret.chunk_content[0]  # first line contents word
         assert len(ret[ret.relevance < 0.65]) == 0
 
-    def test_dict_as_model(self, openai_api_key, reranking_model=None, embedding_model=None):
```

Comment:
this test was created for testing relevance model, let's keep it.
we can rename it to  test_relevance

Reply:
I re-introduced with some changes, @ea-rus.